packages/ckeditor5-media-embed/src/mediaembedediting.js

```diff
@@ -181,6 +184,7 @@ export default class MediaEmbedEditing extends Plugin {
 				const url = modelElement.getAttribute( 'url' );
 
 				return createMediaFigureElement( writer, registry, url, {
+					elementName,
```

Comment:
Please add the missing JSDoc `@param {String} [options.elementName]` to the following functions descriptions, just like you did for `getViewElement()`:

- `createMediaFigureElement()` in `packages/ckeditor5-media-embed/src/utils.js`
- `modelToViewUrlAttributeConverter()` in `packages/ckeditor5-media-embed/src/converters.js`
- `getMediaViewElement()` in `packages/ckeditor5-media-embed/src/mediaregistry.js`

In `modelToViewUrlAttributeConverter()` you can also add an optional short description for this param.

Reply:
Done in 780250c